.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+python:
+  - "3.6"
+dist: trusty
+sudo: required
+# command to install dependencies
+install:
+  - wget -N http://chromedriver.storage.googleapis.com/2.30/chromedriver_linux64.zip -P ~/
+  - unzip ~/chromedriver_linux64.zip -d ~/
+  - rm ~/chromedriver_linux64.zip
+  - sudo mv -f ~/chromedriver /usr/local/share/
+  - sudo chmod +x /usr/local/share/chromedriver
+  - sudo ln -s /usr/local/share/chromedriver /usr/local/bin/chromedriver
+  - pip install .
+  - pip install -r requirements.txt
+# command to run tests
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 
+  
+script: 
+  - set -e
+  - whereis google-chrome-stable
+  - google-chrome-stable --version
+  - whereis chromedriver
+  - pytest -v tests/test_SeleniumPytest.py
```

Comment:
This test doesn't exist for this project, but you can try `make all` instead.

Reply:
done